### PR TITLE
implement `feature_normal_at_point` for trimesh

### PR DESCRIPTION
**Pull request by thefakeplace**
I needed this and couldn't think of any reason it shouldn't be implemented.

### CODE REVIEW

File: src/shape/trimesh.rs [hunk 1]
```diff
@@ -1010,6 +1010,21 @@ impl TriMesh {
             )
         })
     }
+
+    /// Gets the normal of the triangle represented by `feature`.
+    pub fn feature_normal_at_point(
+            &self,
+            feature: FeatureId,
+            _point: &Point<Real>,
```
[the remark this answers is not in the document]
Comment (Vrixyz) [reply src/shape/trimesh.rs:1018]: I imagine itś to keep consistency with the `Shape` method.

**Comment by thefakeplace** [comment]
oh cool my first rust pull request on this account :)

**Comment by thefakeplace** [comment]
oh word i accidentally clicked closed, later boys

**Comment by Vrixyz** [comment]
> oh word i accidentally clicked closed, later boys

No problem, I'm reopening :)

File: src/shape/shape.rs [hunk 2]
```diff
@@ -1130,6 +1130,14 @@ impl Shape for TriMesh {
         Real::frac_pi_4()
     }
 
+    fn feature_normal_at_point(
+        &self,
+        feature: FeatureId,
+        point: &Point<Real>,
+    ) -> Option<Unit<Vector<Real>>> {
+        self.feature_normal_at_point(feature, point)
```
Comment (Vrixyz) [inline src/shape/shape.rs:1138]: I don´t understand this, this will result in an infinitely recursing loop.

**Pull request merged**